api/apsystem.ts

```diff
@@ -29,12 +30,14 @@ export default class ActivityPubSystem {
   store: Store
   modCheck: ModerationChecker
   fetch: FetchLike
+  hookSystem: HookSystem
 
   constructor (publicURL: string, store: Store, modCheck: ModerationChecker, fetch: FetchLike = globalThis.fetch) {
     this.publicURL = publicURL
     this.store = store
     this.modCheck = modCheck
     this.fetch = fetch
+    this.hookSystem = new HookSystem(store, fetch)
```

Comment:
The hooksystem should be passed into the constructor in case we wanted to mock it.